.github/workflows/sdk.yml

```diff
@@ -275,6 +275,14 @@ jobs:
           path: |
             python/openmldb/test/pytest.xml
 
+      - name: upload python coverage to codecov
+        uses: codecov/codecov-action@v2
+        with:
+          name: python coverage
```

Comment:
name should be same with others: `coverage`

Reply:
Done